### PR TITLE
Eventing 6.6.0 editorial updates

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/eventing/pages/eventing-Terminologies.adoc [hunk 1]
```diff
@@ -64,14 +64,14 @@ When a source bucket is deleted, all deployed functions associated with this sou
...
+If the handler code directly modifies documents in the source bucket, the Eventing Service will suppress the mutation back to the Handler making the mutation. 
+When implementing multiple Functions it is possible to create infinite recursions, however the Eventing Service by default will prevent deploying Functions that would result in recursion loops.  It should be noted that not all recursion loops can be detected nor are all recursion loops wrong -- the default recursion checks can be disabled. For more detail on cyclic generation of data changes, refer to xref:troubleshooting-best-practices.adoc#cyclicredun[Bucket Allocation Considerations].
 
 *Metadata Bucket*
 
-The Metadata Bucket stores artifacts (or configuration documents) such as, DCP streams, worker allocations, timer information, along with internal checkpoint information.
+The Metadata Bucket stores artifacts (or configuration documents), such as DCP streams, worker allocations, timer information, along with internal checkpoint information.
```
Comment (jon-strabala) [inline modules/eventing/pages/eventing-Terminologies.adoc:72]: The Metadata Bucket stores artifacts (or configuration documents) that contain information about DCP streams, worker allocations, timers, and internal checkpoint information.

File: modules/eventing/pages/eventing-Terminologies.adoc [hunk 2]
```diff
@@ -133,7 +133,7 @@ The deploy operation activates an Eventing function or handler. Eventing functio
 
 This operation activates a handler. Source validations are performed, and only valid handlers can be deployed. Deployment transpiles the code and creates the executable artifacts. The source code of an activated (or deployed and running) handler cannot be edited. Unless a handler is in deployed state, it will not receive or process any events. Deployment of a Function creates necessary metadata, spawns worker processes, calculates initial partitions, and initiates check-pointing of DCP stream to processes. 
 
-Deployment for DCP observer (or Feed Boundary) has three variations:
+Deployment for DCP observer (or Feed Boundary) has two variations:
```
Comment (jon-strabala) [inline modules/eventing/pages/eventing-Terminologies.adoc:136]: Yes two (2) ... I must have been think about pause

File: modules/eventing/pages/eventing-examples-cancel-overwrite-timer.adoc [hunk 3]
```diff
@@ -7,45 +7,45 @@
...
 
-** *Test 2*: The control document mutated in such a way that any existing Timer with the reference of the control documents id (meta.id) is canceled, this has no effect if the Timer created has already fired.
+** *Test 2*: The control document is mutated in such a way that any existing Timer with the reference of the control document's id (meta.id) is canceled. This has no effect if the Timer created has already fired.
 
-** *Test 3*: The control document mutated in such a way that any existing Timer with the reference of the control documents id (meta.id) is overwritten, this is equivalent to canceling a timer is it already exists and then creating a new Timer that will fire approximately 60 seconds in the future.
+** *Test 3*: The control document is mutated in such a way that any existing Timer with the reference of the control document's id (meta.id) is overwritten. This is equivalent to canceling a timer as it already exists, and then creating a new Timer that will fire approximately 60 seconds in the future.
```
Comment (jon-strabala) [inline modules/eventing/pages/eventing-examples-cancel-overwrite-timer.adoc:13]: ** *Test 3*: The control document is mutated in such a way that any existing Timer with the reference of the control document's id (meta.id) is overwritten. This is equivalent to canceling a timer that already exists, and then creating a new Timer that will fire approximately 60 seconds in the future.

( Iprefer "that" over "as it"

File: modules/eventing/pages/eventing-examples-cancel-overwrite-timer.adoc [hunk 4]
```diff
@@ -7,45 +7,45 @@
...
 
 *Preparations (Common)*
 
-For this example, three buckets 'source', 'target', and 'metadata', are required (note the metadata bucket for Eventing can be shared with other Eventing functions). Make all three buckets with minimum size of 100MB.
+For this example, three buckets 'source', 'target', and 'metadata', are required. (Note the metadata bucket for Eventing can be shared with other Eventing functions.) Make all three buckets with minimum size of 100MB.
```
Comment (jon-strabala) [inline modules/eventing/pages/eventing-examples-cancel-overwrite-timer.adoc:16]: Maybe lose the parens (Note ... functions.)

File: modules/eventing/pages/eventing-faq.adoc [hunk 5]
```diff
@@ -125,7 +125,7 @@ Yes.
 Functions can be "paused, edited and resumed" without losing a single mutation; as such continuity is maintained. Note this sequence is similar to the operations of "undeploy, edited, and deployed with a feed boundary ‘From now’", however in this later case you can lose mutations.
 
 
-* How to perform Functions lifecycle operations from CI/CD?
+* How do I perform a Function's lifecycle operations from CI/CD?
 +
 To perform Functions lifecycle operations from CI/CD, refer to the https://docs.couchbase.com/server/6.5/cli/cbcli/couchbase-cli-eventing-function-setup.html[CLI Eventing] section.
```
Comment (jon-strabala) [inline modules/eventing/pages/eventing-faq.adoc:130]: should we reference /6.5/ in the URL or /6.6/ or /current/ I don't know if they are live ....

File: modules/eventing/pages/eventing-language-constructs.adoc [hunk 6]
```diff
@@ -328,7 +328,7 @@ function OnDelete(meta,options) {
 }
 ----
 
-Note pre 6.6.0 argument syntax is still fully supported, albeit you will not be able to differentiate deletion from expiry.
+Note that the pre-6.6.0 argument syntax is still fully supported, but you will not be able to differentiate deletion from expiry.
```
Comment (jon-strabala) [inline modules/eventing/pages/eventing-language-constructs.adoc:331]: deletion from expiration.

File: modules/eventing/pages/eventing-timers.adoc [hunk 7]
```diff
@@ -151,21 +151,7 @@ The additional overall delay is an implementation artifact and may change betwee
...
-
-* Creating timers in the future (as in one hour+) in an otherwise idle system can result in a growing number of metadata bucket operations which can eventually block mutations for a given Eventing function. In 6.5.X a user space work around can be accomplished via a second cooperative Function.  
-+
-The severity is governed by:
-+
-** The number of vBuckets holding an active timer. Therefore if there are only a few timers in the future the issue may not be noticeable or materialize.
-** Whether an Eventing timer has fired recently on a vBucket (which clears the issue for the given vBucket on a per function basis). Therefore systems with lots of near term timer activity will not experience this issue even if timers are scheduled far into the future. 
-
-All of these limitations are removed in release 6.6.0
+In Couchbase Server 6.6.0 and later, Eventing timers offer the following improvements over previous versions:
```
Comment (jon-strabala) [inline modules/eventing/pages/eventing-timers.adoc:154]: The limitations in prior versions do indeed mean something and are important.  I assume this information was kept in the 6.5 documentation tree - correct?

File: modules/eventing/pages/eventing-faq.adoc (hunk 5 above)
Comment (amarantha-k) [reply modules/eventing/pages/eventing-faq.adoc:130]: In such cases, we use xrefs...  xref:cli:cbcli/couchbase-cli-eventing-function-setup.adoc

File: modules/eventing/pages/eventing-timers.adoc (hunk 7 above)
Comment (amarantha-k) [reply modules/eventing/pages/eventing-timers.adoc:154]: yes, information related to 6.5 was kept in the 6.5 branch, and info related to 6.6 is in the 6.6 branch

**Pull request merged**